lua/nvim-tree.lua

```diff
@@ -462,8 +465,13 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
   },
   update_focused_file = {
     enable = false,
-    update_root = false,
-    ignore_list = {},
+    update_root = {
+      enable = false,
+      ignore_list = {},
+    },
+    exclude = function()
```

Comment:
Please use false here, rather than a function, following the convention of similar. This keeps the config small.

Please put a new "function" definiton in `ACCEPTED_TYPES` so that validation may occur.

You'll need to update help as well.

Reply:
Fixed